Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -215,13 +216,20 @@ class TouchableOpacity extends React.Component<Props, State> {
       ...eventHandlersWithoutBlurAndFocus
     } = this.state.pressability.getEventHandlers();
 
+    const {disabled} = this.props;
```

Comment:
I think we want this value of `disabled` to match the one in line 141, so that they're kept in sync. Perhaps this constant should be defined above and used in both places.